src/acc/libsmm_acc/libcusmm/kernels/cusmm_dnt_largeDB2.py

```diff
@@ -55,20 +54,21 @@ def promising_parameters(m, n, k, gpu):
                         rmax = (m + tm - 1) // tm
 
                         # Miniumum number of threads required to have one thread per tile
+                        # i.e., cover the result matrix
                         min_threads = cmax * rmax
                         if threads < min_threads:
                             continue
                         if (min_threads < (threads - 32)):
                             continue  # heuristic: too many threads unused during calculation
 
-                        for w in range(4, (k + 1)/2, 2):  # heuristic: only even numbers
+                        for w in range(4, (k + 1)/2, 2):  # heuristic: even numbers yield better performance
```

Comment:
warning: the divison here could yield floating point numbers in Python 3, not sure whether that was intended.
Point being: the behavior may differ between Python 2 and 3. Possibly add `from __future__ import division` as a first import to prevent that.